docs/pages/getting-started/docker-compose.mdx

```diff
@@ -0,0 +1,217 @@
+---
+title: Getting started with Teleport using Docker Compose
+description: How to get started with Teleport Open Source Edition using Docker Compose locally.
+h1: Get started with Docker Compose
+---
+
+This section will cover:
+
+- Getting started with a local Teleport cluster using docker compose.
+- Using Teleport with OpenSSH, ansible and Teleport's native client, `tsh`.
+
+## Prerequisites
+
+- Teleport v(=teleport.version=) Open Source or Enterprise.
+- Docker v(=docker.version=) or later and [`docker-compose`](https://docs.docker.com/compose/install/) v(=docker.compose.version=) or later.
+
+```bash
+docker-compose version
+# docker-compose version (=docker.compose.version=), build unknown
+
+docker version
+#Client: Docker Engine - Community
+# Version:           (=docker.version=)
+```
+
+## Step 1/3. Start teleport demo lab
+
+Let's use `docker-compose` to start Teleport demo lab - a configured local cluster:
+
+```bash
+# Download the quick start file from our Github repo
+curl -Lso teleport-lab.yml https://raw.githubusercontent.com/gravitational/teleport/v(=teleport.version=)/docker/teleport-lab.yml
```

Comment:
Also, this doesn't work without you also cloning the repo:

```
vagrant@linux:~/docker$ curl -Lso teleport-lab.yaml https://raw.githubusercontent.com/gravitational/teleport/sasha/demo/docker/teleport-lab.yaml
vagrant@linux:~/docker$ ls -l
total 4
-rw-rw-r-- 1 vagrant vagrant 1846 Jul 13 14:53 teleport-lab.yaml
vagrant@linux:~/docker$ docker-compose -f teleport-lab.yaml up
ERROR: build path /home/vagrant/docker/sshd either does not exist, is not accessible, or is not a valid URL.
```

I guess this is why you want to pre-build the image via release so that people don't need to build them locally ourselves?

Reply:
yes, exactly

Reply:
yes, this is where I'd need your help - could not figure out how to add it to the pipeline